corehq/apps/app_manager/models.py

```diff
@@ -1042,6 +1042,7 @@ class FormBase(DocumentSchema):
     is_release_notes_form = BooleanProperty(default=False)
     enable_release_notes = BooleanProperty(default=False)
     session_endpoint_id = StringProperty(exclude_if_none=True)  # See toggles.SESSION_ENDPOINTS
+    respect_relevancy = BooleanProperty(default=False, exclude_if_none=True)
```

Comment:
This is a very non-descriptive name. But it is too early in the morning to think of a better one.

Reply:
I opted for `respect-relevancy` in `suite.xml` because this flag ties to `relevany` condition for the forms and disable them. It might seem non descriptive at first but I think it's not if someone is familiar with concept of relevancy conditions in CommCare.